README.rst

```diff
@@ -28,16 +28,16 @@ If you have deployed client-side STF and want to test it locally, you can run th
 
     ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml
 
-OR
+OR::
 
     ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml --tags "<one of the tags from above>"
 
-To get a summarised output of the tests at the end, whitelist the provided logging callback.::
+To get a summarised output of the tests at the end, use the provided logging callback::
 
     ANSIBLE_CALLBACK_WHITELIST=custom_logger ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml
 
-The logging callback will summarise the tests run on each node and whether they passes or failed.
-The callback will only report on the status of tasks that have a name beinging with "[Test]".
+The logging callback will summarise the tests run on each node and whether they passed or failed.
+The callback will only report on the status of tasks that have a name beginning with ``[Test]``.
 
 Note::
```

Comment:
```suggestion
Note::

```

Reply:
Actually, it should be ``.. note::`` without a newline. The ``::`` is a mistake originally